packages/react-vis/package.json

```diff
@@ -63,15 +63,16 @@
     "@babel/preset-react": "^7.0.0",
     "@babel/register": "^7.0.0",
     "babel-eslint": "^10.1.0",
+    "babel-jest":"^25.5.1",
     "babel-plugin-module-resolver": "^4.0.0",
     "babelify": "^10.0.0",
     "browserify": "^14.3.0",
     "canvas-prebuilt": "^1.6.11",
     "enzyme": "^3.11.0",
     "enzyme-adapter-react-16": "^1.15.2",
+    "jest":"^25.5.4",
     "jsdom": "^9.9.1",
     "node-sass": "^4.9.3",
-    "nyc": "^13.0.1",
```

Comment:
According to [nyc docs](https://www.npmjs.com/package/nyc): 

> If you use jest or tap, you do not need to install nyc. Those runners already have the IstanbulJS libraries to provide coverage for you.

Reply:
😄